app/build.gradle

```
@@ -36,13 +36,15 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.2.0'
+
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+
     implementation 'com.google.android.material:material:1.2.1'
 
-    implementation 'androidx.core:core-ktx:1.2.0'
-    testImplementation 'junit:junit:4.+'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    implementation 'androidx.core:core-ktx:1.3.2'
+    testImplementation 'junit:junit:4.13.1'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
```

Тебе сука меня уговаривать не придется. Где тут код какой-нибудь
```suggestion
    androidTestImplementation 'androidx.prolaps.espresso:espresso-core:3.3.0'
```

---

А лан оставь так))

---

Оп заанрезолвил как тебе такое илон маск